src/Api/IApiMarker.cs

```diff
@@ -0,0 +1,6 @@
+namespace Api;
+
+public interface IApiMarker
+{
+    
```

Comment:
What is this for?

Reply:
for WebApplicationFactory used in test projects to find its assembly